scss/docs/example.scss

```diff
@@ -2,10 +2,36 @@
 
 @import '../vanilla';
 @include vf-u-baseline-grid;
-@include vf-u-theme-toggle;
+@include vf-b-button;
+@include vf-p-segmented-control;
+@include vf-p-forms-inline;
 
 // stylelint-disable selector-max-type -- examples can use type selectors
 body {
   margin: 1rem;
 }
 // stylelint-enable selector-max-type
+
+.p-example-controls {
+  @include vf-transition($property: #{transform, box-shadow, visibility}, $duration: snap);
+  align-items: center;
+  background-color: var(--vf-color-background-alt);
+  bottom: 0;
+  box-shadow: $box-shadow--deep;
+  display: flex;
+  flex-flow: row wrap;
+  gap: $sp-unit;
+  justify-content: space-between;
+  left: 0;
+  padding: $spv--x-small $sph--x-small;
+  position: fixed;
+  right: 0;
+
+  .u-theme-toggle__button {
```

Comment:
If we need a class for it, I guess it would be good to follow the convention of BEM and call it `p-example-controls__button`, not pretend it's a "utility".

Also, as the label is set in JS, maybe we can remove it alltogether and capitalise the first letter in JS (although it's nice to keep this presentational code in CSS).